src/main/kotlin/MyFancyDiscordClient.kt

```diff
@@ -20,10 +37,30 @@ class MyFancyDiscordClient(private val botToken: String) {
         val channel = event.message.serverChannel ?: return
         val author = event.message.author ?: return
 
+        // メッセージをフィルタリングする
         if (author.bot) return
         if (respondChannel.indexOf(channel.id) == -1) return
+        if (!event.message.content.startsWith(prefix)) return
+
+        try {
+            // コマンドを実行する
+            val result: RespondMessage = CommandExecutor.parseAndExec(event.message, prefix)
```

Comment:
```suggestion
            val messageWithoutPrefix: String = event.message.content.removePrefix(prefix)
            val result: RespondMessage = CommandExecutor.parseAndExec(
                messageWithoutPrefix, 
                event.message.channel
                event.message.author, 
            )
```
こういう呼び出しの関数にしたほうがいいのでは